packages/mobile/src/components/lineup-tile/LineupTileAccessStatus.tsx

```diff
@@ -50,17 +51,24 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
   },
   usdcPurchase: {
     backgroundColor: palette.specialLightGreen
+  },
+  unlocked: {
+    borderRadius: spacing(3),
+    paddingVertical: spacing(0.5),
+    paddingHorizontal: spacing(2.5)
```

Comment:
hm we shouldn't have .5 spacing, might wanna double-check with design?

Reply:
Yeah this was mainly a case of trying to work within the old system. Instead I reworked this to use harmony spacing instead 👍